spark/src/main/scala/geotrellis/spark/ingest/Ingest.scala

```diff
@@ -74,13 +73,15 @@ object Ingest {
     val rasterRdd = new RasterRDD(tiledRdd, rasterMetaData)
 
     def buildPyramid(zoom: Int, rdd: RasterRDD[K]): List[(Int, RasterRDD[K])] = {
-      if (zoom > 1) {
+      if (zoom >= 1) {
         rdd.persist(cacheLevel)
         sink(rdd, zoom)
         val pyramidLevel@(nextZoom, nextRdd) = Pyramid.up(rdd, layoutScheme, zoom)
         pyramidLevel :: buildPyramid(nextZoom, nextRdd)
-      } else
+      } else {
+        sink(rdd, zoom)
```

Comment:
Hmmm...yeah I don't know what the lowest zoom level users would want to store at. I guess 0 makes sense.